lib/samples/create_planar_and_geodetic_buffers/create_planar_and_geodetic_buffers_sample.dart

```diff
@@ -0,0 +1,246 @@
+    // Configure the graphics overlay for the geodetic buffers.
+    _geodeticOverlay.renderer = SimpleRenderer(
+      symbol: SimpleFillSymbol(
+        style: SimpleFillSymbolStyle.solid,
+        color: Colors.green,
+        outline: SimpleLineSymbol(
+          style: SimpleLineSymbolStyle.solid,
+          color: Colors.black,
+          width: 2.0,
+        ),
+      ),
+    );
+    _geodeticOverlay.opacity = 0.5;
+
+    // Configure the graphics overlay for the planar buffers.
+    _planarOverlay.renderer = SimpleRenderer(
+      symbol: SimpleFillSymbol(
+        style: SimpleFillSymbolStyle.solid,
+        color: Colors.red,
+        outline: SimpleLineSymbol(
+          style: SimpleLineSymbolStyle.solid,
+          color: Colors.black,
+          width: 2.0,
+        ),
+      ),
+    );
+    _planarOverlay.opacity = 0.5;
+
+    // Configure the graphics overlay for the tapped points.
+    _pointOverlay.renderer = SimpleRenderer(
+      symbol: SimpleMarkerSymbol(
+        style: SimpleMarkerSymbolStyle.cross,
+        color: Colors.white,
+        size: 14.0,
+      ),
+    );
+
+    // Add the overlays to the map view.
+    _mapViewController.graphicsOverlays.addAll(
+      [
+        _geodeticOverlay,
+        _planarOverlay,
+        _pointOverlay,
+      ],
+    );
+
+    // Create a map with the topographic basemap style and set to the map view.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+    _mapViewController.arcGISMap = map;
+
+    // Set the ready state variable to true to enable the sample UI.
+    setState(() => _ready = true);
+  }
+
+  void onTap(Offset screenPoint) {
+    // Capture the tapped point and convert it to a map point.
+    final mapPoint = _mapViewController.screenToLocation(screen: screenPoint);
+    if (mapPoint == null) return;
+
+    // Create a geodetic buffer around the tapped point at the specified distance.
+    final geodeticGeometry = GeometryEngine.bufferGeodetic(
+      geometry: mapPoint,
+      distance: _bufferRadius,
+      distanceUnit: LinearUnit(unitId: LinearUnitId.miles),
+      maxDeviation: double.nan,
+      curveType: GeodeticCurveType.geodesic,
+    );
+    // Create and add a graphic to the geodetic overlay.
+    final geodeticGraphic = Graphic(geometry: geodeticGeometry);
+    _geodeticOverlay.graphics.add(geodeticGraphic);
+
+    // Create a planar buffer around the tapped point at the specified distance.
+    final planarGeometry = GeometryEngine.buffer(
+      geometry: mapPoint,
+      distance: _bufferRadius * 1609.344, // Convert miles to meters.
+    );
+    // Create and add a graphic to the planar overlay.
+    final planarGraphic = Graphic(geometry: planarGeometry);
+    _planarOverlay.graphics.add(planarGraphic);
+
+    // Create and add a graphic to the point overlay.
+    final pointGraphic = Graphic(geometry: mapPoint);
+    _pointOverlay.graphics.add(pointGraphic);
+  }
+
+  // Clear the graphics overlays.
+  void clear() {
```

Comment:
I think we need to override the dispose() method to clear up and release the graphics overlay also.